astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+
+In the root directory of the repo you'll find a Docker compose file (docker-compose.yml) and an environment variables configuration file (.env). Assuming you have Docker installed on your machine, you can stand up FusionAuth up on your machine with:
+
+```
+docker-compose up -d
+```
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](docs/v1/tech/installation-guide/kickstart). When FusionAuth comes up for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to a certain initial state.
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by docker-compose by executing `docker-compose down -v`, then re-run `docker-compose up -d`.
+</Aside>
+
+FusionAuth will be initially configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example username is `user@fusionauthexample.com` and the password is `userpassword`.
+* Your admin username is `admin@fusionauthexample.com` and the password is `adminpassword`.
+* The base URL of FusionAuth `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+### Create your .Net 7 Application
+
+Now you are going to create a .Net application using C#. While this section builds a simple .Net web application, you can use the same configuration to integrate your existing .net application with FusionAuth.
+
+You'll use [Razor Pages](https://learn.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-7.0&tabs=visual-studio) and .Net 7. This application will display common information to all users. There will also be a secured area, only available to an authenticated user. The [full source code](https://github.com/fusionauth/fusionauth-example-dotnet-guide) is available if you want to download it and take a look.
+
+<Aside type="note">
+If you simply want to run the application , there is a completed version in the 'complete-application' directory.  You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+dotnet publish -r osx-x64
+dotnet run
+```
+
+Then view the website at the following location `http://localhost:5000`.
+</Aside>
+
+Since the complete application already exists in the `complete-application` directory, you will need to delete all files in the `compete-application` directory or choose a directory other than `fusionauth-quickstart-dotenet-web` to build the new application. 
+
+Next open your command-line shell (i.e. Terminal on a Mac or Powershell on Windows) and navigate to the directory of your choice.  Then create a new web application using the `dotnet` CLI and go to that directory:
+
+```shell
+dotnet new webapp -o complete-application
```

Comment:
Hmmm. I don't see a good way around this because you include the Program.cs which has a hardcoded namespace of the name of the application. Any suggestions?

Reply:
I think I can fix this.  I will get to work on it.

Reply:
done